drivers/usb/udc/Kconfig.kinetis

```
@@ -0,0 +1,19 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_UDC_KINETIS := nxp,kinetis-usbd
+
+config UDC_KINETIS
+	bool "Kinetis USB device controller driver"
+	imply UDC_WORKQUEUE
+	default $(dt_compat_enabled,$(DT_COMPAT_UDC_KINETIS))
```

This should use the `DT_HAS_..._ENABLED` helpers.

```suggestion
	default y
	depends on DT_HAS_NXP_KINETIS_USBD_ENABLED
```

---

Thanks, I was lazy and forgot.